src/main/java/com/google/api/gax/grpc/ApiCallSettings.java

```diff
@@ -159,11 +163,12 @@ public Builder setRetrySettingsBuilder(RetrySettings.Builder retrySettingsBuilde
     }
 
     /**
-     * Builds an instance of the containing class.
+     * Builds an instance of the containing class. This operation is unsupported for ApiCallSettings
```

Comment:
The wording on this method and the Builder class should be tweaked some, because the concrete implementing classes inherit this documentation. Also, if someone is holding a reference to ApiCallSettings.Builder but the instance is a derived class, calling build() is valid. Think of this case:

ApiCallSettings.Builder builder = PageStreamingCallSettings.newBuilder(...);
ApiCallSettings settings = builder.build(); // valid call; in an IDE, if a user hovers on this, they will see the doc from ApiCallSettings.build()

So the documentation should definitely call out that build() can't be called on an ApiCallSettings.Builder, but it can be called on derived class instances.